[2025-01-24T19:01:17.393+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T19:01:17.404+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2022-04-10T00:00:00+00:00 [queued]>
[2025-01-24T19:01:17.427+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2022-04-10T00:00:00+00:00 [queued]>
[2025-01-24T19:01:17.428+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-24T19:01:17.542+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): echo_hello_world> on 2022-04-10 00:00:00+00:00
[2025-01-24T19:01:17.546+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1200) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T19:01:17.615+0000] {standard_task_runner.py:72} INFO - Started process 1205 to run task
[2025-01-24T19:01:17.547+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'coleta_dados_abertos_cnpj', 'echo_hello_world', 'scheduled__2022-04-10T00:00:00+00:00', '--job-id', '560', '--raw', '--subdir', 'DAGS_FOLDER/coleta_dados_abertos_cnpj/dag.py', '--cfg-path', '/tmp/tmp5f3ydg55']
[2025-01-24T19:01:17.650+0000] {standard_task_runner.py:105} INFO - Job 560: Subtask echo_hello_world
[2025-01-24T19:01:17.690+0000] {task_command.py:467} INFO - Running <TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2022-04-10T00:00:00+00:00 [running]> on host ee70781c29c7
[2025-01-24T19:01:17.744+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Corvaum' AIRFLOW_CTX_DAG_ID='coleta_dados_abertos_cnpj' AIRFLOW_CTX_TASK_ID='echo_hello_world' AIRFLOW_CTX_EXECUTION_DATE='2022-04-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-10T00:00:00+00:00'
[2025-01-24T19:01:17.745+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T19:01:17.754+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-24T19:01:17.755+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello, Airflow!"']
[2025-01-24T19:01:17.761+0000] {subprocess.py:99} INFO - Output:
[2025-01-24T19:01:17.762+0000] {subprocess.py:106} INFO - Hello, Airflow!
[2025-01-24T19:01:17.763+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-24T19:01:17.777+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T19:01:17.777+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=coleta_dados_abertos_cnpj, task_id=echo_hello_world, run_id=scheduled__2022-04-10T00:00:00+00:00, execution_date=20220410T000000, start_date=20250124T190117, end_date=20250124T190117
[2025-01-24T19:01:17.823+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T19:01:17.831+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-24T19:01:17.832+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
